content/en/blog/2025/roadmap/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Istio Roadmap for 2025-2026"
+description: Looking ahead to what's next for Istio.
+publishdate: 2025-07-20
+attribution: "Mitch Connors, for the Istio TOC"
+keywords: [Istio,roadmap,ambient]
+---
+
+Over the next 12 months, we will focus on improving parity between sidecar mode and ambient mode, providing a supported path for sidecar users to migrate to the ambient data plane when they are ready.  We will also revamp our contributor experience, simplifying the process for proposing and implementing new features, and giving recognition to our most valuable contributors. We plan to grow our ecosystem by adding or updating Istio’s integration to various popular cloud native projects and build more case studies for Istio.
+
+## Looking Back
+
+Since 2023, the Istio project has been focused on maturity and innovation, solidifying our position as the best service mesh regardless of sidecars or ambient. These efforts included our CNCF graduation in July 2023, the promotion of Telemetry API and Gateway API to Stable in Istio 1.22, and the promotion of ambient mode to Stable in Istio 1.24. As part of Istio ambient mode reaching GA, we have observed more and more users exploring and adopting it, some of the users are net new Istio users, while others are users of Istio sidecars.  Some of them ran ambient in production and spoke about their experiences at KubeCon EU in April this year. These efforts have made Istio the service mesh of choice for cloud native developers around the world, and we have been excited to accept first code contributions from 154 people in the past 12 months.
+
+## 2025 Themes
+
+### Sidecar to ambient migration
+
+With the promotion of Istio ambient mode to Stable, the world’s fastest service mesh is also the world’s most efficient, and is now easier to operate than ever. With graduation, we’ve seen a substantial increase in interest, and a corresponding number of requests for a comprehensive migration guide for existing sidecar users.  While our previous efforts to stabilize ambient mode have been targeted at new Istio users, it is clear that the time has come to provide an onramp for our existing user base to migrate to ambient mesh.  While the technical foundations for this migration have been in place for some time (and some brave users have migrated on their own), we will be making new investments in automated migration testing, tooling to assess your readiness to migrate, and documentation to guide users every step of the way.
```

Comment:
Don't know if I'm comfortable promising automated migration testing. That's a nontrivial lift

Reply:
agreed - maybe lean into 'rollback safe' along with the tooling part ?

Reply:
"we will be making new investments in tooling to assess your readiness to migrate,  rollback-safe interoperability, and documentation to guide users every step of the way.'

Reply:
I'm fine changing the verbiage, but we don't advertise "support" for anything we don't have automated tests for.  Automated migration tests are a requirement as I understand it.